test/index.js

```diff
@@ -14,7 +15,7 @@ exec('node .', function(err, stdout, stderr) {
             throw err;
         }
 
-        var expected = exp.split('\n');
+        var expected = platform() == 'win32' ? exp.split('\r\n') : exp.split('\n');
```

Comment:
We might be able to use [`os.EOL`](https://nodejs.org/api/os.html#os_os_eol) here instead to simplify.

```js
var os = require('os')
//...
var expected = exp.split(os.EOS);
```